01xx - Mobile Gateway Activity Benchmark 10x Boost.md

```diff
@@ -0,0 +1,62 @@
+**WIP Helium Improvement Proposal (HIP): Mobile Gateway Activity Benchmark 10x Boost**
+ 
+*HIP Number:* 1XX
+ 
+*Title:* Mobile Radio Data Transfer Requirements for Proof-of-Coverage (POC) Rewards
+ 
+*Author:* ElonTusk007, Capc0m (Max), fizzy99
+ 
+*Status:* Draft
+ 
+*Created:* Feb 1, 2024
```

Comment:
I know this is still a WIP but would be helpful to clean up this front matter to match something like this:

```
# HIP 109: Hex Boosting by Deployment

- Author(s): [Andy Zyvoloski](https://github.com/heatedlime) &  [Max Gold](https://github.com/maxgold91)
- Start Date: 2024-01-27
- Category: Economic, Technical
- Original HIP PR: <!--- TBD -->
- Tracking Issue: <!--- TBD -->
- Vote Requirements: veMOBILE Holders
```

when ready

Reply:
Yep, doing that as we speak. :)